### ISSUE TITLE
Crash when toggling to disable effects in windows profile

### ISSUE THREAD

**Issue by yokuyuki** [SpoinkyNL/Artemis#238]
I get the below crash when I toggle the "Enable effect" radio button to disabled in Windows Profile. In older versions, I toggled it as a quick way of re-enabling my profile after a system resume.

~~~
An unexpected error occurred in Artemis.

This type of CollectionView does not support changes to its SourceCollection from a thread different from the Dispatcher thread.
____________________________________________________
System.NotSupportedException

Message:
This type of CollectionView does not support changes to its SourceCollection from a thread different from the Dispatcher thread.

Stack Trace:
   at System.Windows.Data.CollectionView.OnCollectionChanged(Object sender, NotifyCollectionChangedEventArgs args)
   at System.Collections.ObjectModel.ObservableCollection`1.OnCollectionChanged(NotifyCollectionChangedEventArgs e)
   at System.Collections.ObjectModel.ObservableCollection`1.ClearItems()
   at Artemis.ViewModels.Profiles.ProfileEditorViewModel.LoadProfiles()
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at Artemis.Managers.DeviceManager.ReleaseActiveKeyboard(Boolean save)
   at Artemis.Managers.LoopManager.Render()
   at Artemis.Managers.LoopManager.LoopTimerOnElapsed(Object sender, ElapsedEventArgs elapsedEventArgs)

TargetSite:
Void OnCollectionChanged(System.Object, System.Collections.Specialized.NotifyCollectionChangedEventArgs)

Source:
PresentationFramework

HResult:
-2146233067
~~~

**Comment by SpoinkyNL** [comment]
Hey, thanks for the report. Will fix this in tomorrow's update hopefully along with a fix for resuming after sleep

**Comment by SpoinkyNL** [comment]
Hey, fixed in 1.7. I sadly didn't get around to fixing the after-sleep bug but I'll leave #204 open for that issue
